wrappers/javascript/packages/anoncreds-nodejs/src/NodeJSAnoncreds.ts

```diff
@@ -706,6 +1090,22 @@ export class NodeJSAnoncreds implements Anoncreds {
     return this.objectFromJson(this.nativeAnoncreds.anoncreds_key_correctness_proof_from_json, options)
   }
 
+  public w3cCredentialOfferFromJson(options: { json: string }): ObjectHandle {
+    return this.objectFromJson(this.nativeAnoncreds.anoncreds_credential_offer_from_json, options)
+  }
+
+  public w3cCredentialRequestFromJson(options: { json: string }): ObjectHandle {
+    return this.objectFromJson(this.nativeAnoncreds.anoncreds_credential_request_from_json, options)
+  }
```

Comment:
As these just call `anoncreds_credential_offer_from_json` I think we shouldn't add these new methods. Duplicating all methods doesn't make sense to me, and I think we should keep the duplication of methods only to the methods where it's actually needed